topics/global-game-jam/index.md

```diff
@@ -0,0 +1,12 @@
+---
+display_name: Global Game Jam
+created_by:  Susan Gold, and Gorm Lai, and Ian Schreiber
+aliases: globalgamejam, ggj, ggj2009, ggj2010, ggj2011, ggj2012, ggj2013, ggj2014, ggj2015, ggj2016, ggj2017, ggj2018, ggj2019, ggj2020, ggj2021, ggj2022, ggj2023, ggj2024
```

Comment:
Test complaining about too many aliases. We may want to raise this limit or add an exception to this test: https://travis-ci.org/github/explore/builds/347067233?utm_source=github_status&utm_medium=notification